### DISCUSSION TITLE
Welcome to PyKOB Discussions!

### DISCUSSION THREAD

**Discussion by AESilky** [MorseKOB/PyKOB#200] [General]
## 👋 Welcome!
  We’re enabling Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leskerr** [comment]
I'm a founding member of this project. I've been doing American Morse since 2002 and my goal is to keep the sound of Morse alive.

**Reply by jchausler** [reply]
I've been interested in land line telegraphy for about 25 years now and a user of MorseKOB for close to 15 years.  My purpose here is to act as a beta tester.  Although a former software engineer by trade, the world has moved pretty much beyond my skill set and so I leave the development to those with much more skill :-)

**Comment by microPaul** [comment]
I'm a user of MorseKOB and look forward to following this project.
